### PR TITLE
complete ticket-9

### PR DESCRIPTION
**Pull request by yuridevweb**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -213,3 +213,57 @@ describe('GET /api/articles/:article_id/comments', () => {
...
+        })
+      })
+  })
+  test('Status:400, when invalid article_id is passed', () => {
+    const commentData = {
+      username: 'rogersop',
+      body: 'British following properties',
+    }
+    return request(app)
+      .post('/api/articles/333/comments')
```
Comment (cavalierswain) [inline __tests__/app.test.js:246]: would this be a 400 - bad request or a 404 - not found? The id is correct it just doesnt exist yet!

**Comment by cavalierswain** [comment]
looks good! 
Have a look at the comment in the testing!

File: __tests__/app.test.js (hunk 1 above)
Comment (yuridevweb) [reply __tests__/app.test.js:246]: I reckon 404 would be more appropriate

Comment (yuridevweb) [reply __tests__/app.test.js:246]: Actually I think 400 is absolutely fine. We are creating a new comment at this endpoint, and  we can't create a comment with invalid  article_id. So basically we are providing bad data. Is that right?

**Pull request merged**